CODEOWNERS

```diff
@@ -0,0 +1 @@
+/src/ansys/templates/python/solution/ @ansys/Solution-Maintainers
```

Comment:
Should there also be a line for `src/ansys/templates/python/osl_solution` ?

Reply:
Me -> 🤡

Reply:
Changes made